### DISCUSSION TITLE
Searchable option picker/dropdown

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#4001] [Ideas]
Hello,

In some of my dropdown has too many options(over 5000) and I don't want to show all, the "Autocomplete" only allow static filter, it will be much better if it can support dynamic filter. My current workaround is put a separated “Text Field” to allow user input the filter keywords.

![image](https://user-images.githubusercontent.com/29649651/149098447-41005464-2a5c-4461-9479-1c1027c528fd.png)

**Comment by roafhtun** [comment]
Were you able to find any other solution ?

**Comment by joebudi** [comment]
https://github.com/ConorWebb96/bb-type-ahead

**Reply by roafhtun** [reply]
Thanks, for some reason in my hosted Budibase plugins field only show up in the design tab, whenever I change to design or publish that specific plugin field is not visible. Happens with the one you linked, as well as with this one https://github.com/CSLTech/budibase-autocomplete that I tried before.

Edited: Updated the budibase installation, now they show up. Time to figure out how to make the plugin work.

Edited: I am getting error CouchDB error: missing with the plugin.
